README.md

```diff
@@ -6,16 +6,15 @@ The project uses a virtual environment to isolate package dependencies. To creat
 
 ### On macOS and Linux
 ```bash
-$ source setup.sh
-```
-### On Windows (Using Git Bash)
-```bash
-$ source setup.sh --windows
+$ poetry install
 ```
+You'll also need to clone a new .env file from the .env.tempalate to store local configuration options. This is a one-time operation on first setup:
+
+$ cp .env.template .env  # (first time only)
 
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
+Once the all dependencies have been installed, start the Flask app in development mode within the poetry environment by running:
 ```bash
-$ flask run
+$ poetry run flask run
 ```
 
```

Comment:
It's worth having a line somewhere in the README just to say that running the app with vagrant is an option

Reply:
Added line